docs/changelog.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 1.0.2 
+## 1.0.3
```

Comment:
Given this adds new SHALL-level requirements for issuers, I think it's at least a minor version bump per https://semver.org. Like, issuers who were valid against the 1.0 spec might fail our tests after these changes. (Though, clients who work with 1.0 will all still work after this change, so I can be convinced it's a minor version bump.)

Reply:
I'm ok with that, but this is mostly a deployment requirement: someone could update their TLS config without touching their SHC implementation and (re-)become compliant. The larger question is: what are we asking validation app to do? Reject JWKS if offered at a non-secure enough endpoint? Or is that more the job of, say, a VCI directory auditor to contact issuers to upgrade their deployments.

Reply:
> this is mostly a deployment requirement: someone could update their TLS config without touching their SHC implementation

Yeah, that's fair. I'd say the "implementation" of SMART Health Cards Spec covers quite a bit of the stack, including (as you say) code and deployment configuration. On principle, I think the major semver number on the protocol should reflect spec changes that cause previously-valid implementations to become invalid (whether the required changes mean writing new code, deploying a new version of openssl, setting permissions on a file, etc).

> what are we asking validation app to do? Reject JWKS if offered at a non-secure enough endpoint? Or is that more the job of, say, a VCI directory auditor to contact issuers to upgrade their deployments.

For the validation app, I think we'd log an error if a JWKS if the endpoint doesn't support any of the cyphersuites we require here (i.e., we'd say "the issuer is not compliant with the spec".) Downstream, the VCI directory auditing process would evaluate this (or some heuristic approximating this, like using a well-crafted openssl client config that the VCI directory publishes) when adding new endpoints, and periodically after that.

Reply:
It will probably be hard for SHC apps and validation tools to check this, because the TLS details are typically handled by  libraries that don't offer visibility of these details.

Reply:
Bumped minor version, as suggested